.github/workflows/pr-tests.yml

```diff
@@ -231,8 +207,6 @@ jobs:
         run: rustup component add rust-src --toolchain nightly-2024-08-01-x86_64-unknown-linux-gnu
       - name: Install riscv target
         run: rustup target add riscv32imac-unknown-none-elf --toolchain nightly-2024-08-01-x86_64-unknown-linux-gnu
-      - name: Install pilcom
-        run: git clone https://github.com/0xPolygonHermez/pilcom.git  && cd pilcom && npm install
       - uses: taiki-e/install-action@nextest
       - name: Run slow tests
         # Number threads is set to 2 because the runner does not have enough memory for more.
```

Comment:
Same here, is this still the case without pilcom?

Reply:
removed